qute.ls/com.redhat.qute.ls/src/test/java/sandbox/TestItCountQute.java

```diff
@@ -0,0 +1,47 @@
+package sandbox;

```

Comment:
Maybe accidentally committed?

Reply:
No I use those templates in sandbox to test qute with the qute engine.

Reply:
I see my bad.